<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fzqjReportClass</class>
 <widget class="QMainWindow" name="fzqjReportClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>547</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>325</width>
    <height>547</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>手动上报</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="local_lon_name">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>29</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>本机经度：</string>
    </property>
   </widget>
   <widget class="QLabel" name="local_lat_name">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>78</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>本机纬度：</string>
    </property>
   </widget>
   <widget class="QLabel" name="local_alt_name">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>127</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>本机高度：</string>
    </property>
   </widget>
   <widget class="QLabel" name="target_pc_name">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>274</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>目标俯仰：</string>
    </property>
   </widget>
   <widget class="QLabel" name="target_dis_name">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>323</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>目标距离：</string>
    </property>
   </widget>
   <widget class="QLabel" name="target_type">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>176</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>目标类型：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_report_send">
    <property name="geometry">
     <rect>
      <x>194</x>
      <y>380</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>上报</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_local_lon_value">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_local_lat_value">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_local_alt_value">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_target_Az_value">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>220</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_target_pc_value">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>270</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_target_dis_value">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>320</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cbx_target_type_list">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>170</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="currentText">
     <string>不明</string>
    </property>
    <item>
     <property name="text">
      <string>不明</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>直升机</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>固定翼无人机</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>旋翼无人机</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>车</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>鸟</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>空飘球</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>大型无人机</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>空飘物</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="target_Az_name">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>225</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>目标方位：</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>325</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
